frontend/src/views/Project/RolePage/components/RolePermissionsSection/ProjectRoleModifySection.utils.ts

```diff
@@ -175,6 +185,14 @@ export const rolePermission2Form = (permissions: TProjectPermission[] = []) => {
         if (canEdit) formVal[subject as ProjectPermissionSub.Member]![0].edit = true;
         if (canCreate) formVal[subject as ProjectPermissionSub.Member]![0].create = true;
         if (canDelete) formVal[subject as ProjectPermissionSub.Member]![0].delete = true;
+
+        if (subject === ProjectPermissionSub.Cmek) {
```

Comment:
Yah to avoid like this, let's keep the CMEK actions as one separate enum and an if condition just for that. Because the more like these others devs has to loop up multiple places to make changes

Reply:
refactored for separate enum